### PR TITLE
Ports status display smooth scrolling, icon, and font from /tg/station

### PR DESCRIPTION
**Pull request by Generalcamo**
This ports smooth scrolling, the new icon, and the new font for status displays from /tg/station.
### Asset Licenses
The following assets that **have not** been created by myself are included in this PR:

| Path | Original Author | License |
| --- | --- | --- |
| icons/obj/status_display.dmi | [DrDiasyl](https://github.com/DrDiasyl) /tg/station) | CC-BY-SA |

### CODE REVIEW

**Comment by Generalcamo** [comment]
!review

**Comment by desvenlafaxine** [comment]
I feel like the biohazard screen doesn't fit our setting, as it has a news anchor reporting on the situation.

Also, I don't know if the default screen should be that bright, isn't it meant to be when the screen is off? On a note, maybe it should also have a SCC logo.

**Comment by Generalcamo** [comment]
Ignore icondiffbot, it's broken and is not showing the animations correctly. The news anchor is for the untouched "entertainer" state.

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by FluffyGhoster** [comment]
This is failing the SDMM linter

File: code/game/machinery/status_display.dm [hunk 1]
```diff
@@ -81,96 +90,149 @@
...
 
 	switch(mode)
 		if(STATUS_DISPLAY_BLANK)	//blank
+			remove_messages()
 			return 1
 		if(STATUS_DISPLAY_TRANSFER_SHUTTLE_TIME)				//emergency shuttle timer
 			if(evacuation_controller)
 				if(evacuation_controller.is_prepared())
 					message1 = "-ETD-"
+					message2 = "Test"
```
Comment (Arrow768) [inline code/game/machinery/status_display.dm:104]: Is that intentional or a leftover?

**Review by Arrow768 — CHANGES_REQUESTED** [review]

File: code/game/machinery/status_display.dm (hunk 1 above)
Comment (Generalcamo) [inline code/game/machinery/status_display.dm:104]:
```suggestion
```

Comment (Generalcamo) [reply code/game/machinery/status_display.dm]: Leftover from /tg/

**Review by Geevies — APPROVED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**